typing/oprint.ml

```diff
@@ -309,6 +309,24 @@ and print_out_type_1 ppf =
       pp_print_space ppf ();
       print_out_type_1 ppf ty2;
       pp_close_box ppf ()
+  | Otyp_functor (lab, id, (p, fl), ty) ->
+      pp_open_box ppf 0;
+      print_arg_label ppf lab;
+      pp_print_string ppf "(module ";
+      print_ident ppf id;
+      pp_print_string ppf " : ";
+      print_ident ppf p;
+      let first = ref true in
+      List.iter
```

Comment:
This is fine. For future reference, you can also use `List.iteri (fun i (s, t) -> if i = 0 then ... ) fl`.

Reply:
This was the code already existing for printing packages, as it is now an auxiliary function untouched by this PR it might be better to to improvement in already existing code in a separate PR